### ISSUE TITLE
Hyperparameters used for the results in the paper

### ISSUE THREAD

**Issue by Martin36** [UKPLab/fever-2018-team-athene#37]
I am trying to reproduce the results of the sentence retriever model, but can't seem to get the accuracy to the same level as in the paper. I tried using the hyperparameters from the `config.py` file, but the accuracy is significantly lower than what is mentioned in the paper and the training stops after about 5 epochs due to no increase in the performence. In the paper no hyperparameters are mentioned. Do you still have information regarding the hyperparameters from the paper?
